src/styles/globals.scss

```diff
@@ -7,14 +7,36 @@ body,
 html {
   height: 100%;
   line-height: 1.5;
+  background-color: var(--background);
 }
 
 * {
   font-family: "kanit", sans-serif;
 }
 
+:root {
+  --background: #F9F9FB;
+  --card: #ffffff;
+  --low-contrast: #D9D9D9;
+  --button: #000000;
+  --accent: #131313;
+  --text: #000000;
+  --pistachio: #94FFA9;
+}
+
+.dark {
+  --background: #131313; 
+  --card: #242222;
+  --low-contrast: #575050;
+  --button:  var(--low-contrast);
+  --accent: #a59d9d;
+  --text: #dedede;
+  --pistachio: #00550e;
+}
```

Comment:
I would expect this to be handled via tailwind somehow. Isnt it?

Reply:
Unfortunately no 😔

Reply:
What about this https://tailwindcss.com/docs/dark-mode ? That looks pretty convincing it could work

Reply:
By the way,  `<AuthProvider>` also uses "use client" inside already and wraps the `<Layout>` 😅

Reply:
And I also made a few tests 
<img width="774" alt="image" src="https://github.com/GLObus303/sportujspolu/assets/47262567/4cfb4242-e607-4386-90b4-c000ac3029a1">

<img width="310" alt="image" src="https://github.com/GLObus303/sportujspolu/assets/47262567/3a60afb0-960e-407a-b397-546461799fda">

1) console.log logs into VSCode terminal and not browser so it was on the server side. 
2) use react hook inside server component - and still I get server components error:

<img width="873" alt="image" src="https://github.com/GLObus303/sportujspolu/assets/47262567/5d17a847-7ad8-4d81-a8d4-9e16778378f7">


```
./src/app/page.tsx
ReactServerComponentsError:

You're importing a component that needs useEffect. It only works in a Client Component but none of its parents are marked with "use client", so they're Server Components by default.
Learn more: https://nextjs.org/docs/getting-started/react-essentials

   ╭─[/Users/jo/sportujspolu/src/app/page.tsx:1:1]
 1 │ import { useEffect } from 'react';
   ·          ─────────
 2 │ 
 3 │ import { Events } from '../components/Events';
 4 │ import { getAllEvents } from '../api/events';
   ╰────

Maybe one of these should be marked as a client entry with "use client":
  ./src/app/page.tsx
```